framework/include/base/MooseFunctor.h

```diff
@@ -604,15 +606,15 @@ template <typename T>
 typename FunctorBase<T>::ValueType
 FunctorBase<T>::operator()(const FaceArg & face_in, const StateArg & state) const
 {
-  const auto face = checkFace(face_in);
+  checkFace(face_in);
 
-  if (_clearance_schedule.count(EXEC_ALWAYS))
-    return evaluate(face, state);
+  if (_clearance_schedule.empty())
```

Comment:
Definitely, or change the name. Because now the `_clearance_schedule` almost appears to behave the opposite of its name ... if there is no clearance schedule, which would imply to me that you should never clear the cache, but you instead always evaluate seems counter-intuitive

Reply:
Yeah, sorry for the lack of comments. Just some background here: On a test case with 500k cells (3D channel) the count operations on the clearence schedule actually showed up in the profiles. With this change I could save 5.5% runtime on 200 iterations. So it decreased the runtime from 163 to 154 s on a single core. Also, this results in slight acceleration for the other functor-based codes as well. I might rename it, just wanted you to know my tivation.

Reply:
I ended up keeping the `clearance_schedule` name with adding an `_always_evaluate` member that is set to `true` if somebody assigns `EXEC_ALWAYS`. And we use that variable for the early return.